### DISCUSSION TITLE
Define what content is allowed in contributions?

### DISCUSSION THREAD

**Discussion by 320kg** [breezy-weather/breezy-weather#1410] [Ideas]
The title is self explanatory. Please note that I am unfamiliar with Kotlin and Java and have a limited understanding of the Android building pipeline (yet, willing to learn to contribute) and I am just wondering what would and wouldn't get accepted. 

For example, are native (arch specific) libraries fucking welcome here? What about raw binary parsers? Custom interfaces for integrating into other programs?

**Comment by papjul** [comment]
You can start by reading [CONTRIBUTE.md](https://github.com/breezy-weather/breezy-weather/blob/main/CONTRIBUTE.md) to get a general idea.

You can also check the [list of what's not allowed](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).

Then open one (and only one) idea discussion for each thing you want to implement, and we will tell you for each, if we want it implemented in the project.

Note that a `ContentProvider` is planned as part of #863 so anything that doesn't fit could be implemented elsewhere.

**Reply by papjul** [reply]
Also, please fill in the GitHub website template when you open an idea ;)
